src/blueprint/from_config.rs

```diff
@@ -393,10 +373,71 @@ fn process_path(
   ))
 }
 
+fn process_field_within_type(
+  field_name: &str,
+  remaining_path: &[String],
+  type_info: &config::Type,
+  is_required: bool,
+  config: &Config,
+  invalid_path_handler: &dyn Fn(&str, &[String]) -> Valid<Type>,
+) -> Valid<Type> {
+  if let Some(next_field) = type_info.fields.get(field_name) {
+    if needs_resolving(next_field) {
+      return Valid::<Type>::validate_or(
+        Valid::fail("Inline can be done on path with resolvers".to_string()).trace(field_name),
```

Comment:
```suggestion
        Valid::fail("Inline can't be done on path with resolvers".to_string()).trace(field_name),
```
We can specify the exact resolver which cause us to bail out.